libs/libjson/_internal/Source/JSONNode.h

```diff
@@ -161,7 +161,7 @@ class JSONNode {
 
 
     json_string as_string(void) const json_nothrow json_read_priority;
-    int as_int(void) const json_nothrow json_read_priority;
```

Comment:
libjson has stopped the development as is unmaintained now. Does this answer? :-)